src/batched/KokkosBatched_InverseLU_Serial_Impl.hpp

```diff
@@ -0,0 +1,196 @@
+#ifndef __KOKKOSBATCHED_INVERSELU_SERIAL_IMPL_HPP__
+#define __KOKKOSBATCHED_INVERSELU_SERIAL_IMPL_HPP__
+
+
+/// \author Vinh Dang (vqdang@sandia.gov)
+
+#include "KokkosBatched_Util.hpp"
+#include "KokkosBatched_Trsm_Decl.hpp"
+#include "KokkosBatched_Trsm_Serial_Impl.hpp"
+
+
+namespace KokkosBatched {
+  namespace Experimental {
+    ///
+    /// Serial Impl
+    /// =========
+
+    ///
+    /// InverseLU no piv
+    ///
+
+#if                                                     \
+  defined(__KOKKOSBATCHED_INTEL_MKL__) &&               \
+  defined(__KOKKOSBATCHED_INTEL_MKL_BATCHED__) &&       \
+  defined(__KOKKOSBATCHED_INTEL_MKL_COMPACT_BATCHED__)
+    template<>
+    template<typename AViewType,
+             typename WViewType>
+    KOKKOS_INLINE_FUNCTION
+    int
+    SerialInverseLU<Algo::InverseLU::CompactMKL>::
+    invoke(const AViewType &A,
+           const WViewType &W) {
+      typedef typename AViewType::value_type vector_type;
+      //typedef typename vector_type::value_type value_type;
+
+      const int
+        m = A.extent(0),
+        n = A.extent(1);
+
+      static_assert(is_vector<vector_type>::value, "value type is not vector type");      
+      static_assert(vector_type::vector_length == 4 || vector_type::vector_length == 8, 
+                    "AVX, AVX2 and AVX512 is supported");
+      assert(m==n);
+
+      const MKL_COMPACT_PACK format = vector_type::vector_length == 8 ?  MKL_COMPACT_AVX512 : MKL_COMPACT_AVX;
+
+      int r_val = 0;
+      if (A.stride_0() == 1) {
```

Comment:
Christian does not like to use stride_0. Plz use stride(0).

Reply:
Fixed.